src/installer.ts

```diff
@@ -76,15 +77,105 @@ export async function getJuliaVersions(versionInfo): Promise<string[]> {
     return versions
 }
 
-export function getJuliaVersion(availableReleases: string[], versionInput: string, includePrerelease: boolean = false): string {
+export function getProjectFile(projectInput: string = ""): string {
+    let projectFile: string = ""
+
+    // Default value for projectInput
+    if (!projectInput) {
+        projectInput = process.env.JULIA_PROJECT || "."
```

Comment:
What will happen here if:
1. `JULIA_PROJECT` is defined, but it is set to the empty string.
2. `JULIA_PROJECT` is defined, and it is nonempty, but it consists only of whitespace.

Reply:
```js
> JULIA_PROJECT = ""; JULIA_PROJECT || "."
'.'
> JULIA_PROJECT = " "; JULIA_PROJECT || "."
' '
```

Reply:
I've added additional tests around using `JULIA_PROJECT`